src/components/HelpDialog.vue

```diff
@@ -17,17 +17,22 @@
       >
         <div class="column full-height">
           <q-list>
-            <q-item
-              v-for="(page, i) of pagedata"
-              :key="i"
-              clickable
-              v-ripple
-              active-class="selected-item"
-              :active="selectedPage === page.name"
-              @click="selectedPage = page.name"
-            >
-              <q-item-section>{{ page.name }}</q-item-section>
-            </q-item>
+            <template v-for="(page, i) of pagedata" :key="i">
```

Comment:
でかいfor loopでの変数`i`は、変数の重複によるかなりバグを生みやすいので避けられると嬉しいです。
（特にOSSだと他人のコードに書く形になるので）

この場合、page_indexとかがわかりやすいと思います！
```suggestion
            <template v-for="(page, page_index) of pagedata" :key="page_index">
```

Reply:
他はcamelCaseなので、こちらもcamelCaseの方が良さそうです。

Reply:
修正しました